### ISSUE TITLE
Thermostat: Eco mode adds 3°C, target temperature set (former: F@H things/channels)

### ISSUE THREAD

**Issue by kjoglum** [ruebox/openhab2-addons#46]
Not sure if this thread is alive, but have a couple of comments/questions related to the F@H binding.

First of all, appreciate the great work!

Related to the binding:
- For the thermostats: Is it not possible to set the desired temperature? See target temperature and measured temperature as values (Paper UI)
- ECO mode (thermostats) adds 3 degrees to the target temperature, not subtracting
- Any tips for adding Welcome M, touch panels (4.3" panels have integrated thermostats), actions etc?

**Comment by lassem** [comment]
Welcome M I am not sure about. Pretty sure you need to have the IP gateway in order to get something out of that.

I actually have one, but I think it's a old KNX layover and I am only using the outdoor part connected to my Free@Home system. Apparently these two components are interchangeable between the KNX and Free@Home variants.

Anywho, I can't see it in my web-interface, but it works with the 7" display.

**Comment by kjoglum** [comment]
Yes, same here, everything works as supposed to with the 7" display within Free@Home. Just curious if possible to implement (without IP gateway) the camera/audio module. 

Have had some situations where camera has been triggered (photo) without the doorbell being triggered. Would be interesting to know if there are some "hidden channels" which could be used in OpenHab. I.e. if motion triggered, take photo (as it seems has happened within F@H, although just in rare occasions).

Also, the thermostat as part of the 4.3" touch panels would be a nice feature to include.

**Comment by ruebox** [comment]
@kjoglum Are you still facing this issue?
Could you please provide me the deviceId of your device? 

@lassem Are you also facing this issue? If I am remind my correctly, you are also running thermostats. Isn't it.

> * For the thermostats: Is it not possible to set the desired temperature? See target temperature and measured temperature as values (Paper UI)
> 
> * ECO mode (thermostats) adds 3 degrees to the target temperature, not subtracting
